README.md

```diff
@@ -1,11 +1,9 @@
 # Hito 0
 
 # Problema a solventar
-Evitar que la comida se ponga mala en el frigorífico y pedir automáticamente cuando se acaben productos básicos.
+Comparador de vuelos 
 
 # Descripción 
-Es muy común comprar cosas en el supermercado y semanas más tarde ver que estaban en el frigorífico y se han puesto malas. Pues esta aplicación tendrá almacenadas una serie de alimentos  y una fecha de caducidad aproximada, por lo tanto cuando se acerque la fecha y el alimento siga en el frigorífico el programa avisará para evitar que el alimento se ponga malo.
-
-Además, este programa permitirá realizar compras periódicas y, cuando se acabe un alimento o se ponga malo, sugerirá al usuario si desea que se vuelva a comprar y lo hará automáticamente
+Esta aplicación se basará en la monitorización de un vuelo determinado que el usuario elija, y el mismo usuario será quien decida si quiere que se le avise cuando el precio baje de X, o si desea recibir una gráfica con la evolución del precio del vuelo durante el tiempo deseado.
```

Comment:
Os insisto en que debéis tratar de resolver un problema, porque sólo vais a poder hacer una aplicación razonable si es así. ¿Qué problema resuelve al usuario saber cómo ha evolucionado el precio del vuelo en una gráfica?

Reply:
Pues lo he puesto porque he comprobado por experiencia que un vuelo de 80€ mirado a las 11 de la mañana del martes, a las 11 de la mañana del miércoles puede valer 67€. Yo considero que es útil y que es algo que además de no estar desarrollado (yo lo he buscado y no lo he encontrado) permite ahorrar un dinero si quieres que te avise o simplemente que busques el vuelo solo a las horas que ya ha comprobado el programa que el vuelo suele bajar

Reply:
Como hemos dicho en clase, 
1. Debes resolver un problema
2. El problema debe necesitar una cierta lógica de negocio para solucionarse.

Por favor, revisa el tutorial sobre el significado ce "Lógica de negocio" y el guión, donde damos pistas sobre qué tipo de problemas (y soluciones) sirven y no sirven (en general y en esta asignatura). También se ha hablado en clase.